### ISSUE TITLE
NEWS -- Let's keep Termination going!

### ISSUE THREAD

**Issue by Fred-Barclay** [Fred-Barclay/Termination#75]
Hi everyone,
If you're reading this you've probably been linked here from a comment of mine on [platformio-atom-ide-terminal](https://github.com/platformio/platformio-atom-ide-terminal/). Unfortunately it seems that pio-terminal will no longer be actively developed: https://github.com/platformio/platformio-atom-ide-terminal/issues/514#issuecomment-384241678 and https://github.com/platformio/platformio-atom-ide-terminal/issues/543.

This is sad news for us for two reasons: (a) this is a loss to the Atom community as a whole as pio-terminal seems to have easily been the single most popular terminal plugin for Atom, and (b) pio-terminal was our upstream and we relied heavily on it over the past 1,5 years.

Moving forward, it's my hope to keep Termination going as strong as ever (stronger, actually :laughing: ) -- but with the community's backing! 
Anyone is welcome to contribute. Pull requests, suggestions and ideas, even bug reports are all wanted and needed! Ideally, over time termination can be backed by a community of collaborators with direct push access instead of just me merging pull requests and muddling through the best I can. :grin: 

 Plus, I'm not a coffeescript hacker, and for the forseeable future, I can't be. If we want to keep Termination going I need collaborators (or folks willing to gradually come on board and take over) with more experience than I have.

Hopefully we can pick up where pio-termination leaves off and keep providing one of the best terminal experiences in Atom.
Cheers!
Fred

TL; DR: Platformio-terminal is no longer maintained, I hope that Termination can pick up from here so we all can have a great tool for Atom, but need the community's help to do so.
